src/ClientCredentialsOptions.cs

```diff
@@ -3,21 +3,21 @@
 
 namespace Laserfiche.Oauth.Api.Client
 {
-    public class ClientCredentialsOptions : IClientCredentialsOptions
+    public class ClientCredentialsOptions
     {
         [JsonProperty("csid")]
```

Comment:
The JsonProperty names should be updated to match the new property names on the dev console

Reply:
Yes, thanks for spotting it. I will update the code.

Reply:
Fixed. I will change this class to match the TS counterpart.